assured-workloads-example/main.tf

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  aw_consumer_folder_id = one(
+    [for resource in google_assured_workloads_workload.primary.resources : resource.resource_id
+    if resource.resource_type == "CONSUMER_FOLDER"]
+  )
```

Comment:
Same question I had in another topic regarding setting directly here instead of pulling from the resource

Reply:
In this case I believe it's not possible because the Folder ID will be known after its creation.